tests/cloner_test.go

```diff
@@ -0,0 +1,133 @@
+package tests
```

Comment:
Maybe now is the time to add test area dirs under _tests/_? Eg. 
```
tests/cloner/
tests/importer/
tests/controller/
```
This may be too unit-test like so I am open to other suggestions. I think down the road it will be a mess to have every e2e/functional test directly under _tests/_.

Reply:
So have many test files are you expecting to be here. kubevirt which is significantly larger than us has about 20 e2e test files at this point. That will likely grow on their end. I am not expecting more than maybe half a dozen to a dozen files for us. import/clone/upload/various. So I don't think it will be that bad to had them all in this directory.

Reply:
That's fine and it's your choice. Sub-dirs can always be added later if desired.